torchrl/collectors/collectors.py

```diff
@@ -1884,28 +1887,35 @@ class MultiSyncDataCollector(_MultiDataCollector):
     trajectory and the start of the next collection.
     This class can be safely used with online RL sota-implementations.
 
+    note:: Python requires multiprocessed code to be instantiated within a
+    `if __name__ == "__main__":` block. See https://docs.python.org/3/library/multiprocessing.html
+    for more info.
```

Comment:
This doesn't render well
![image](https://github.com/pytorch/rl/assets/25529882/5a00eaa4-4806-4b4f-a6c5-5432d7dfd2c5)


```suggestion
    .. note:: Python requires multiprocessed code to be instantiated within a main guard:

            >>> if __name__ == "__main__":
            ...     # Create your collector here

        See https://docs.python.org/3/library/multiprocessing.html for more info.

```